statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -6,6 +6,8 @@
 ----------
 Lütkepohl (2005) New Introduction to Multiple Time Series Analysis
 """
+from __future__ import annotations
```

Comment:
Hi @bashtage, `__future__.annotations` is only avaialbe in Python 3.7 or newer. Does this mean `statsmodels` will drop Python 3.6 support?

https://github.com/statsmodels/statsmodels/blob/main/setup.py#L357

Reply:
I noticed this change in our CI which runs tests using the dev version of statsmodels using Python 3.6:

https://github.com/mlflow/mlflow/runs/3111502285?check_suite_focus=true#step:11:44

Reply:
3.7 is the target for the next release, which is aligned with NumPy and pandas, among others.

Reply:
FWIW NumPy is already 2 releases into 3.7 minimum (1.20 and 1.21)

Reply:
@bashtage Thanks for the quick reply! Got it, we'll update our CI to use Python 3.7.

Reply:
Are you planning to update this line?

https://github.com/statsmodels/statsmodels/blob/86e1c85e922fe72dd83a4e7869aa0e52807a3dc6/setup.py#L357

Reply:
Thanks. in #7590

Reply:
Awesome, thanks!